packages/account/src/Components/poi/status/unsupported/detail-component.tsx

```diff
@@ -63,11 +63,11 @@ const DetailComponent = ({
                 const expiration_date =
                     typeof data.expiry_date?.format === 'function' ? data.expiry_date.format('YYYY-MM-DD') : undefined;
                 uploadFile(file, WS.getSocket, {
-                    documentType: document_type,
-                    pageType,
-                    expirationDate: expiration_date,
-                    documentId: data.document_id || '',
-                    lifetimeValid: +(lifetime_valid && !expiration_date),
+                    document_type,
+                    page_type: pageType,
+                    expiration_date,
+                    document_id: data.document_id || '',
+                    lifetime_valid: +(lifetime_valid && !expiration_date),
```

Comment:
Can you please double check as according to the uploader [here](https://github.com/binary-com/binary-document-uploader/blob/7dadc0f922882c6a1bd9f8713b339bfe54aea2e7/src/client.js#L18), `documentType` and other metadata is camelcase